src/main/java/frc/robot/commands/cargo/CargoIntake.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands.cargo;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+
+public class CargoIntake extends Command {
```

Comment:
Could this be an instant command?

Reply:
We could change it to whileheld as an instant command. Would it still need a whenreleased if we did that?

Reply:
Same sentiment applies to all cargo commands

Reply:
you mean a whenreleased to stop the motor, right?